.travis.yml

```diff
@@ -0,0 +1,33 @@
+language: bash
+services: docker
+
+branches:
+  only:
+    - develop
+
+addons:
+  apt:
+    packages:
+      - shellcheck
+
+env:
+  global:
+    - TERRAFORM_VERSION=0.9.3 GT_SITE_SETTINGS_BUCKET=geotrellis-site-staging-config-us-east-1
```

Comment:
I think it would be clearer to list this as:

```yaml
env:
  global:
    - TERRAFORM_VERSION=0.9.3
    - GT_SITE_SETTINGS_BUCKET=geotrellis-site-staging-config-us-east-1
    - secure: OZh5yap7uhylrN11GzoMLywTfkYBISzSfK5v...
    - secure: cjZWJUYgrLGUOKBe7DP2uYJK6eZ0CxqamLhYdXfie5v5R...
```